tests/calc/test_thermo.py

```diff
@@ -190,6 +191,16 @@ def test_parcel_profile_lcl():
     assert_array_almost_equal(prof, true_prof, 2)
 
 
+def test_parcel_profile_lcl_not_monotonic():
+    """Test parcel profile with lcl calculation."""
+    with pytest.raises(InvalidSoundingError):
+        p = np.array([1004., 1000., 943., 925., 928., 850., 839., 749., 700.]) * units.hPa
+        t = np.array([24.2, 24., 20.2, 21.6, 21.4, 20.4, 20.2, 14.4, 13.2]) * units.degC
+        td = np.array([21.9, 22.1, 19.2, 20.5, 20.4, 18.4, 17.4, 8.4, -2.8]) * units.degC
+
+        pressure, temp, dewp, prof = parcel_profile_with_lcl(p, t, td)
```

Comment:
By convention, just dump to `_`:
```suggestion
        _ = parcel_profile_with_lcl(p, t, td)
```

Reply:
Done